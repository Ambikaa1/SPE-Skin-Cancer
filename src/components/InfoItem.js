import React from "react";
import { View, Text, StyleSheet } from "react-native";
import { Ionicons } from '@expo/vector-icons';

const InfoItem = ({ name, index }) => {
    return(
        <View style = {(index === 0) ? styles.containerTop : styles.container}>
            <Text style = {styles.heading}>{name}</Text>
            <Ionicons name = "ios-arrow-forward" size = {17} color = "black" />
        </View>
    );
};

const styles = StyleSheet.create({
    containerTop: {
        marginHorizontal: 10,
        paddingVertical: 10,
        flexDirection: "row",
        justifyContent: "space-between",
        borderBottomColor: "#AAAAAA",
        borderBottomWidth: 0.5,
        borderTopColor: "#AAAAAA",
        borderTopWidth: 0.5,
    },
    container: {
        marginHorizontal: 10,
        paddingVertical: 10,
        flexDirection: "row",
        justifyContent: "space-between",
        borderBottomColor: "#AAAAAA",
        borderBottomWidth: 0.5
    },
    heading: {
        fontSize: 17,
        color: "black",
    },
    icon: {
        color: "black",
    }
});

export default InfoItem;
