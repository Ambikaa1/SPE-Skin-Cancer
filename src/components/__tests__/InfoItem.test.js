import React from 'react';
import renderer from 'react-test-renderer';

import InfoItem from "../InfoItem";

describe('<InfoItem />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<InfoItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
