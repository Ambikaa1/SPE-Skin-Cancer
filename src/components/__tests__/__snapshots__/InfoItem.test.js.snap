// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoItem /> renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "black",
      "borderBottomWidth": 0.5,
      "borderTopColor": "black",
      "borderTopWidth": 0.5,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "marginHorizontal": 10,
      "marginVertical": 2.5,
      "paddingVertical": 10,
    }
  }
>
  <Text
    style={
      Object {
        "color": "black",
        "fontSize": 20,
      }
    }
  />
  <Text />
</View>
`;
